testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -542,22 +542,38 @@ def test_fasta2select_resids(self, tmpdir):
         assert len(sel['reference']) == 30621, self.error_msg
         assert len(sel['mobile']) == 30621, self.error_msg
 
-def test_sequence_alignment():
-    u = mda.Universe(PSF)
-    reference = u.atoms
-    mobile = u.select_atoms("resid 122-159")
-    aln = align.sequence_alignment(mobile, reference)
-
-    assert len(aln) == 5, "return value has wrong tuple size"
-
-    seqA, seqB, score, begin, end = aln
-    assert_equal(seqA, reference.residues.sequence(format="string"),
-                 err_msg="reference sequence mismatch")
-    assert mobile.residues.sequence(
-        format="string") in seqB, "mobile sequence mismatch"
-    assert_almost_equal(score, 54.6)
-    assert_array_equal([begin, end], [0, reference.n_residues])
+class TestSequenceAlignmentFunction:
```

Comment:
```suggestion

class TestSequenceAlignmentFunction:
```
Ok I think we have to look into a pep8 action over diffs (easy enough to do on the whole thing, I'm sure it's doable on diffs).

edit: I've gone ahead and opened a PR to do this.